Skype/UCWA/presence_ref.md

```diff
@@ -1,3 +1,6 @@
+---
+title: Presence resource description 
```

Comment:
Change "resource description" to "reference"

Reply:
fixed.